pubsub_logging/pubsub_handler.py

```diff
@@ -74,8 +75,9 @@ def __init__(self, topic, capacity=DEFAULT_BATCH_NUM,
         self._publish_body = publish_body
         if client:
             self._client = client
-        else:  # pragma: NO COVER
+        else:
             self._client = get_pubsub_client()
+        create_topic(self._client, topic, retry)
```

Comment:
Ah, this is what you said previously:
```
I removed the whole retrying flow upon 404 error, which simplified the code, since I realized that the automatic topic creation is actually useless because a topic without any subscriptions will throw the messages away.
```

Does that still apply?